ui/apps/platform/src/Containers/NetworkGraph/utils/modelUtils.tsx

```diff
@@ -648,3 +652,57 @@ export function getConnectedNodeIds(edges: CustomEdgeModel[], selectedNodeId: st
     }, [] as string[]);
     return connectedNodeIds;
 }
+
+function filterDNSFlows(properties) {
+    return !(
+        (properties.port === 53 || properties.port === 5353) &&
+        properties.protocol === 'L4_PROTOCOL_UDP'
+    );
+}
+
+export function removeDNSFlows(edges: CustomEdgeModel[]): CustomEdgeModel[] {
+    const modifiedEdges: CustomEdgeModel[] = [];
+    edges.forEach((edge) => {
+        const filteredSourceToTargetProperties =
+            edge.data.sourceToTargetProperties.filter(filterDNSFlows);
+        const filteredTargetToSourceProperties =
+            edge.data?.targetToSourceProperties?.filter(filterDNSFlows) || [];
+        const combinedProperties = [
+            ...filteredSourceToTargetProperties,
+            ...filteredTargetToSourceProperties,
+        ];
+
+        if (combinedProperties.length !== 0) {
+            const portProtocolLabel = getPortProtocolEdgeLabel([
+                ...filteredSourceToTargetProperties,
+                ...filteredTargetToSourceProperties,
+            ]);
+            const modifiedData: EdgeData = {
+                sourceToTargetProperties: filteredSourceToTargetProperties,
+                targetToSourceProperties: filteredTargetToSourceProperties,
+                isBidirectional:
+                    filteredTargetToSourceProperties.length !== 0 &&
+                    filteredSourceToTargetProperties.length !== 0,
+                portProtocolLabel,
+                tag: portProtocolLabel,
+            };
+            if (filteredSourceToTargetProperties.length !== 0) {
+                modifiedData.endTerminalType = EdgeTerminalType.directional;
+            } else {
+                modifiedData.endTerminalType = EdgeTerminalType.none;
+            }
+            if (filteredTargetToSourceProperties.length !== 0) {
+                modifiedData.startTerminalType = EdgeTerminalType.directional;
+            } else {
+                modifiedData.startTerminalType = EdgeTerminalType.none;
+            }
+            const modifiedEdge = {
+                ...edge,
+                data: modifiedData,
+            };
+
+            modifiedEdges.push(modifiedEdge);
```

Comment:
Also, this new function should have some unit tests.

Reply:
Yea. The data parsing/transforming is a bit messy here. We could make things cleaner, but I focused more on trying to make it work first. Also, tests would be useful. We can add those too

Reply:
Was getting some funky issue when I tried creating a unit test for this function 
```
FAIL  src/Containers/NetworkGraph/utils/modelUtils.test.ts
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    /Users/schaudhr/go/src/github.com/stackrox/stackrox/ui/node_modules/@patternfly/react-icons/dist/esm/icons/expand-icon.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){import { createIcon } from '../createIcon';
                                                                                      ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at Runtime.createScriptFromCode (../../node_modules/jest-runtime/build/index.js:1728:14)
      at Object.<anonymous> (../../node_modules/@patternfly/react-topology/src/components/TopologyControlBar/TopologyControlBar.tsx:11:1)
      ```
     Mark was helping me debug but we didn't really figure out the actual issue. It's funny because I'm not doing anything related to that icon import, but maybe something is going on internally with PatternFly. I'll try separating my function into its own module to see what happens first. Then I'll try breaking down my function into smaller functions I can test. Maybe that'll help move things forward